engine/dlib/src/dlib/configfile.cpp

```diff
@@ -287,9 +301,9 @@ namespace dmConfigFile
         Expect(context, '=');
         EatBlank(context);
 
-        char value_buf[2048];
-
-        ParseLiteral(context, value_buf, sizeof(value_buf));
+        int value_len = SizeOfLiteral(context) + 1;
+        char* value_buf = new char[value_len];
```

Comment:
One improvement might be to not allocate a new buffer each time ParseEntry is called and instead pass a buffer and length as arguments to ParseEntry and only reallocate the buffer if the value is larger than the buffer.

Reply:
I agree, and I think that is a better approach!

Reply:
Also, why not simply use the upper limit `context.m_BufferSize`?
It's not going to be that huge. and if it is huge, it might also be required!

Reply:
This is a much better approach! One allocation and no need to measure the length of a value!